test/unit/auth/user-import-builder.spec.ts

```diff
@@ -555,6 +601,77 @@ describe('UserImportBuilder', () => {
           new UserImportBuilder(noHashUsers, validOptions as any, userRequestValidator);
       expect(userImportBuilder.buildRequest()).to.deep.equal(expectedRequest);
     });
+
+    it('should return expected request with no multi-factor fields when not available', () => {
+      const noMultiFactorUsers: UserImportRecord[] = [
+        {uid: '1234', email: 'user@example.com', multiFactor: null},
+        {uid: '5678', phoneNumber: '+16505550101', multiFactor: {enrolledFactors: []}},
+      ];
+      const expectedRequest = {
+        users: [
+          {localId: '1234', email: 'user@example.com'},
+          {localId: '5678', phoneNumber: '+16505550101'},
+        ],
+      };
+      const userImportBuilder =
+          new UserImportBuilder(noMultiFactorUsers, validOptions as any, userRequestValidator);
+      expect(userImportBuilder.buildRequest()).to.deep.equal(expectedRequest);
+    });
+
+    it('should ignore users with invalid second factor enrollment time', () => {
+      const invalidMultiFactorUsers: UserImportRecord[] = [
+        {
+          uid: '1234',
+          multiFactor: {
+            enrolledFactors: [
+              {
+                uid: 'enrolledSecondFactor1',
+                phoneNumber: '+16505557348',
+                displayName: 'Spouse\'s phone number',
+                factorId: 'phone',
+                enrollmentTime: 'invalid',
```

Comment:
I expected this to throw. How is it getting silently ignored?

Reply:
Actually it only gets ignored when building the request. The client side errors are cached and will be injected in the response alongside server side response. Check test `should return the expected response for import with client side errors`. I added a test for this.